airbyte-webapp/src/utils/links.ts

```diff
@@ -21,7 +21,7 @@ export const links = {
   namespaceLink: `${BASE_DOCS_LINK}/understanding-airbyte/namespaces`,
   tutorialLink: "https://www.youtube.com/watch?v=Rcpt5SVsMpk&feature=emb_logo",
   statusLink: "https://status.airbyte.io/",
-  recipesLink: "https://airbyte.com/recipes",
```

Comment:
[This page](https://airbyte.com/recipes) now redirects to https://airbyte.com/tutorials causing `:airbyte-webapp:validateLinks` to fail